README.md

```diff
@@ -888,3 +813,444 @@ Optional.Some(3).tap(x  =>  console.log("Hello") ||  x  +  1);
+// This evaluates to 3
+Ok(3).merge();
+
+// This evaluates to 3 too
+Error(3).merge();
+
+// And so does this
+Aborted(3).merge();
+
+// And this returns the wrapped Promise:
+Pending(Promise.resolve(Ok(3))).merge();
+```
+
+#### `Result`'s `getOrElse(value)`
+
+Unwraps the value from a `Result` if it is an `Ok`, else returns the substitute value passed to `getOrElse`.
+
+```javascript
+// This evaluates to 6
+Ok(3).map(x => x * 2).getOrElse(8);
+
+// This evaluates to 8
+Error().map(x => x * 2).getOrElse(8);
+```
+
+#### `Result`'s `replace(value)`
+
+`Result`'s `replace` will replace values wrapped in the `Ok` subtype by the `value` that you pass to `replace`.
+
+```javascript
+// This evaluates to Ok(2)
+Ok(3).replace(2);
+
+// This evaluates to Error(3)
+Error(3).replace(2);
+
+// This evaluates to Aborted(3)
+Aborted(3).replace(2);
+```
+
+#### `Result`'s `satisfies(predicate)`
+
+`Error` or `Aborted` subtypes never `satisfie` any predicates and always evaluate to `false`.
+
+```javascript
+// This evaluates to true
+Ok(3).satisfies(x => x * 3 === 9);
+
+// This evaluates to false
+Ok(3).satisfies(x => x * 2 === 7);
+
+// These also evaluate to false
+Error(3).satisfies(x => x === 3);
+Aborted(3).satisfies(x => x === 3);
+```
+
+#### `Result`'s `map(λ)`
+
+Applies `λ` to your `Result`, returning an `Ok` wrapping the result, if the orignal `Result` is an `Ok`. Otherwise, it evaluates to whatever `Error` or `Aborted` it was applied to.
+
+```javascript
+// Evaluates to Ok(5)
+Ok(4).map(x => x + 1);
+```
+
+#### `Result`'s `mapError(λ)`
+
+Whereas `Result`'s `map` will simply return the original `Error` when applied on an `Error`, `errorMap` transforms an the value wrapped inside the `Error`.
+
+```javascript
+// Evaluates to Error(5)
+Error(4).mapError(x => x + 1);
+
+// Evaluates to 4
+Ok(4).mapError(x => x + 1);
+
+// Evaluates to Aborted('Dead')
+Aborted('Dead').mapError(x => x + 1);
+```
+
+#### `Result`'s `tap(λ)`
+
+Like `map`, but returns the value wrapped by your `Result` unmodified.
+You'd mostly use this if you need the value of your `Result` for side-effects, such as modifying an existing variable that lives outside your `Result` chain, or using the `Result` in an API call, but continuing with the original value, not the return value from the API call.
+
+```javascript
+// Evaluates to 4
+Ok(4).tap(x => x + 1);
+
+// Evaluates to Error
+Error().tap(x => x + 1);
+
+// Evaluates to Aborted
+Aborted().tap(x => x + 1);
+```
+
+#### `Result`'s `flatMap(λ)`
+
+If your `Result` is an `Ok`, `flatMap` maps your `Result` to `λ`, then runs the result through `Result.expect`.
+
+```javascript
+// This evaluates to Error(undefined)
+Ok(null).flatMap(x => x)
+
+// This evaluates to Ok(1)
+Ok(null).flatMap(x => x + 1)
+```
+
+#### `Result`'s `expectMap(λ)`
+
+First maps the `Result` to λ, then runs `expect` on it. The main difference with `flatMap` is that the return value of `λ(value)` is not transformed before being passed to `expect`.
```

Comment:
This is impossible to document. I would remove it from the documentation altogether (the behavior has zero logic behind it), or at least say it's deprecated.